### ISSUE TITLE
Context menu shows but does not work

### ISSUE THREAD

**Issue by a13ph** [minbrowser/min#2452]
- **Min Version:** 1.31.2
- **Operating system:** Windows 10

## To Reproduce
Right click a link then click on `Open in a New Tab` with left mouse button or right mouse button or press enter.

## Expected Behavior
Links open in a new tab.

## Actual Behavior
Links do not open unless I ctrl-click them.

Above is repro specifically for link opening, but none of the context menu entries works currently for me.
Copy, web search, inspector - all fail.
To clarify, context menu itself does show up just fine.
On any webpage, even here on Github new issue page.
Hotkeys do work.
Context menus in other apps work.
Restarting the browser did not help.

There is something weird going on, as I did not have this bug before on that version and I can not recognize or recall of any changes that I have made or incurred which could possibly influence this behavior.

Took me a while to notice that it is a general context menu issue in Min.

**Comment by PalmerAL** [comment]
That is quite weird. Does the menu button in the navigation bar work? What about if you open the searchbar and right-click on the tab URL?

Also, try opening developer > inspect browser > console, then open the context menu again and try to click on something. Do any errors appear in the console?

**Comment by a13ph** [comment]
Thanks for debugging tips, I will try them next time it happens. Right now, system reboot probably has helped, but this bug has definitely happenned to me several times before, on and off, so I will keep my eye on it.
